common/meta/sled-store/src/sled_tree.rs

```diff
@@ -136,15 +134,15 @@ impl SledTree {
                         MetaStorageError::SledError(_e) => {
                             Err(ConflictableTransactionError::Abort(meta_sto_err))
                         }
-                        MetaStorageError::TransactionAbort(_e) => {
-                            Err(ConflictableTransactionError::Abort(meta_sto_err))
-                        }
                         MetaStorageError::TransactionConflict => {
                             Err(ConflictableTransactionError::Conflict)
                         }
                         MetaStorageError::AppError(_app_err) => {
                             Err(ConflictableTransactionError::Abort(meta_sto_err))
                         }
+                        MetaStorageError::SnapshotError(_e) => {
```

Comment:
Why not just use `_` instead of  `_e` ?

Reply:
Hmm... It's a good question that deserves a great answer.